console/console-init/ui/src/modules/address-space/components/AddressSpaceReview/AddressSpaceReview.tsx

```diff
@@ -109,18 +141,82 @@ export const AddressSpaceReview: React.FC<IAddressSpaceReviewProps> = ({
             )}
             {authenticationService && authenticationService.trim() !== "" && (
               <>
-                <GridItem span={5} style={{ marginBottom: 16, marginRight: 5 }}>
+                <GridItem span={5} className={css(style.preview_info_gridItem)}>
                   Authentication Service
                 </GridItem>
                 <GridItem id="preview-addr-plan" span={7}>
                   {authenticationService}
                 </GridItem>
               </>
             )}
+            <br />
+            <br />
+            {customizeEndpoint && (
+              <>
+                <GridItem
+                  span={12}
```

Comment:
Similar, to my previous component, the code here is repetitive and we can avoid that by creating a component and pass values as props.